server/src/test/java/io/crate/integrationtests/GeoShapeIntegrationTest.java

```diff
@@ -72,7 +73,7 @@ public void prepare() throws Exception {
     @Test
     public void testSelectGeoShapeFromSource() throws Exception {
         execute("select shape from shaped where id in (1, 2) order by id");
-        assertThat(TestingHelpers.printedTable(response.rows()), is(
+        assertThat(printedTable(response.rows()), is(
```

Comment:
just fyi, you can use the `assertThat(response).hasRows(` (our `io.crate.testing.Asserts.assertThat`)

Reply:
Thanks, updated to `hasRows`